app/scraper/__init__.py

```diff
@@ -8,6 +8,7 @@
 from threading import Thread
 from app.search import add_to_index
 
+from .s3 import ImageUploader
```

Comment:
Remove now-unused import

Reply:
No, unfortunately. I never figured out how to get the right cookies for the scraper.